lib/ls.js

```diff
@@ -444,10 +466,11 @@ function getExtras (data) {
 }
 
 function makeParseable (data, long, dir, depth, parent, d) {
+  if (data._deduped) return []
   depth = depth || 0
   if (depth > npm.config.get('depth')) return [ makeParseable_(data, long, dir, depth, parent, d) ]
   return [ makeParseable_(data, long, dir, depth, parent, d) ]
-  .concat(Object.keys(data.dependencies || {})
+ .concat(Object.keys(data.dependencies || {})
```

Comment:
I read through the rules and there was no mention of method chains...
I don't know `standard` well enough to call whether it does or doesn't.